app/commands.py

```diff
@@ -167,33 +169,62 @@ def link_inbound_numbers_to_service(self):
 
 
 class PopulateMonthlyBilling(Command):
-        option_list = (
-            Option('-y', '-year', dest="year", help="Use for integer value for year, e.g. 2017"),
+    option_list = (
+        Option('-y', '-year', dest="year", help="Use for integer value for year, e.g. 2017"),
+    )
+
+    def run(self, year):
+        service_ids = get_service_ids_that_need_billing_populated(
+            start_date=datetime(2016, 5, 1), end_date=datetime(2017, 8, 16)
+        )
+        start, end = 1, 13
+        if year == '2016':
+            start = 4
+
+        for service_id in service_ids:
+            print('Starting to populate data for service {}'.format(str(service_id)))
+            print('Starting populating monthly billing for {}'.format(year))
+            for i in range(start, end):
+                print('Population for {}-{}'.format(i, year))
+                self.populate(service_id, year, i)
+
+    def populate(self, service_id, year, month):
+        create_or_update_monthly_billing(service_id, datetime(int(year), int(month), 1))
+        sms_res = get_monthly_billing_by_notification_type(
+            service_id, datetime(int(year), int(month), 1), SMS_TYPE
         )
+        email_res = get_monthly_billing_by_notification_type(
+            service_id, datetime(int(year), int(month), 1), EMAIL_TYPE
+        )
+        print("Finished populating data for {} for service id {}".format(month, str(service_id)))
+        print('SMS: {}'.format(sms_res.monthly_totals))
+        print('Email: {}'.format(email_res.monthly_totals))
+
+
+class BackfillProcessingTime(Command):
+    option_list = (
+        Option('-s', '--start_date', dest='start_date', help="Date (%Y-%m-%d) start date inclusive"),
+        Option('-e', '--end_date', dest='end_date', help="Date (%Y-%m-%d) end date inclusive"),
```

Comment:
It's not inclusive of the end date right

Reply:
this function is. I call `delta.days + 1` on L218 to specifically make it inclusive.

Reply:
apologies, missed the commit message.